Plots/MapProjections/NCL_lcnative_1_lg.py

```diff
@@ -0,0 +1,109 @@
+"""
+NCL_lcnative_1_lg.py
+================
+This script illustrates the following concepts:
+    - Drawing contours over a map using a native lat,lon grid
+    - Drawing filled contours over a Lambert Conformal map
+    - Zooming in on a particular area on a Lambert Conformal map
+    - Subsetting a color map
+    - Using all three Cartopy Lambert projections to find the best fit for visualization
+    - Implementing best practices for choosing contour color scheme
+    
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/lcnative_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/lcnative_1_lg.png
+
+"""
+###############################################################################
+# Import packages:
+
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mticker
+
+
+from geocat.viz import cmaps as gvcmaps
+import geocat.datafiles as gdf
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/pre.8912.mon.nc"), decode_times=False)
+
+# Extract a slice of the data
+t = ds.pre.isel(time=0)
+
+###############################################################################
+# Plot:
+
+plt.figure(figsize=(14, 14))
+
+
+def Plot(row, col, pos, proj, title):
+    '''
+    Args:
+        
+        row (:class: 'int'): 
+            number of rows necessary for subplotting of visualizations
+        col (:class: 'int'): 
+            number of columns necessary for subplotting 
+        pos (:class: 'int'): 
+            position of visualization in m x n subplot
+        proj (:class: 'cartopy.crs'): 
+            which projection to visualize
+        title (:class: 'str'): 
+            center title of respective visualization
+    '''    
+    # Generate axes using Cartopy and draw coastlines
+    projection = proj
+    ax = plt.subplot(row, col, pos, projection=projection)
+    ax.set_extent((28, 57, 20, 47), crs=ccrs.PlateCarree())
+    ax.coastlines(linewidth=0.5)
+    # Import an NCL colormap
+    newcmp = gvcmaps.BlueYellowRed
```

Comment:
If that's intentional to pick a different color map than NCL, please clarify that in the function docstrings.

Reply:
I did mention at the top in the "concepts illustrated" portion that using best practices for choosing color scheme is implemented in the code. However, do you think it would be better to have a comment about this in the code as well, or do one or the other (either comment about it in the code and not at the top, or leave it as is)?

Reply:
Ah, sorry!  I did not notice that. May be you add some information to that concept illustrated about what NCL's original was and what we used in this example.

Reply:
I added a small blurb at the top and linked to the "colors" version of this in the GeoCAT-Examples gallery. I also updated the color scheme to be "Blues_r" because for the "colors" version, we stated using a solid color would be the best route for this type of data

Reply:
It sounds good now except that you need to remove 

```
from geocat.viz import util as gvutil
from geocat.viz import cmaps as gvcmaps
```

because they are not used.